tests/basic_net_vm_import_test.go

```diff
@@ -0,0 +1,101 @@
+package tests_test
+
+import (
+	v2vv1alpha1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1alpha1"
+	"github.com/kubevirt/vm-import-operator/tests/framework"
+	. "github.com/kubevirt/vm-import-operator/tests/matchers"
+	"github.com/kubevirt/vm-import-operator/tests/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var (
+	podType   = "pod"
+	networkID = "123"
+)
+
+type networkedVmImportTest struct{}
+
+var _ = Describe("Networked VM import ", func() {
+	var (
+		f         = framework.NewFrameworkOrDie("networked-vm-import")
+		secret    corev1.Secret
+		namespace string
+		test      = networkedVmImportTest{}
+	)
+
+	BeforeEach(func() {
+		namespace = f.Namespace.Name
```

Comment:
Is the namespace on which the operator is deployed is used also for creating the vm-import resources and the VM ?
If this is the case, perhaps we'd like to make sure they are different, so there aren't any assumptions regarding permissions ?

Reply:
No, it's not. 
`framework.Framework` creates namespace with generated name and prefix given in its constructor:
https://github.com/kubevirt/vm-import-operator/blob/b21b0d4961f4789b51292ea715f1e7186ac96f7e/tests/framework/framework.go#L159